src/web/mod.rs

```diff
@@ -930,20 +659,6 @@ mod test {
         })
     }
 
-    #[test]
-    fn double_slash_does_redirect_and_remove_slash() {
```

Comment:
This is removing the test for the fix from https://github.com/rust-lang/docs.rs/pull/1410 , 
axum doesn't error here, but returns a normal 404.

Reply:
Hmm, can we keep the test even though it passes? I'm a little confused; is this changing behavior from a reidrect to a 404?

Reply:
Yes, this would change behaviour from a redirect to a 404. 
So I _could_ add the test back, changing the expected response to a 404. 

Introducing the same behaviour in the axum routing layer felt not worth the time, of course I could still do it if needed.